gprofiler/main.py

```diff
@@ -723,6 +744,45 @@ def init_pid_file(pid_file: str) -> None:
     Path(pid_file).write_text(str(os.getpid()))
 
 
+# Set limits and return path of the cgroup.
+def set_limits(cpu: float, memory: int) -> str:
+    cgroups = {}
```

Comment:
Any reason to use this dict? Why not 2 variables `cpu_cgroup` and `memory_cgroup`?